core/src/test/java/org/elasticsearch/aliases/IndexAliasesIT.java

```diff
@@ -804,16 +802,17 @@ public void testCreateIndexWithAliasesFilterNotValid() {
     }
 
     // aliases can be added only to indices
```

Comment:
can you remote this comment? I think that the method name explains it well enough?